composer.json

```diff
@@ -4,17 +4,26 @@
     "keywords": ["beanstalkd", "pheanstalk"],
     "license": "Apache-2.0",
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
         "pda/pheanstalk": "^4.0",
-        "symfony/framework-bundle": "^5.4",
-        "symfony/console": "^5.4"
+        "symfony/framework-bundle": "^5.4.34 || ^6.4",
+        "symfony/console": "^5.4.34 || ^6.4"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "symfony/yaml": "^5.4",
-        "symfony/dependency-injection": "^5.4",
-        "symfony/phpunit-bridge": "^5.4",
-        "symfony/console": "^5.4"
+        "symfony/yaml": "^5.4.31 || ^6.4",
+        "symfony/dependency-injection": "^5.4.5 || ^6.4",
+        "symfony/phpunit-bridge": "^5.4 || ^6.4",
+        "symfony/cache": "^5.4 || ^6.4",
+        "symfony/config": "^5.4 || ^6.4",
```

Comment:
Probably want to peg all of these as higher 5.4 versions too, just in case there's some wonky bugs in lower 5.4 versions.

Reply:
Yep, I tested those, they're in 5.4.0. I added them explicitly because if not, they were set to 7.0.